canonical/canonical.com
# Implement the sliding navigation JS

## Done

This work is based around the changes to found [here](https://github.com/canonical/vanilla-framework/pull/5213)
- Implements the logic for the sliding navigation
- Some HTML changes to matches the changes in the aforementioned branch

## QA

- Go to [demo](https://canonical-com-1310.demos.haus/) on mobile view
- Click the 'menu' button, see that it opens the navigation

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-12081
